draft/index.html

```diff
@@ -560,7 +560,7 @@ <h3>Reconciliation Query Responses</h3>
 	    <dt><code>type</code></dt>
 	    <dd>The types of the candidate entity;</dd>
 	    <dt><code>score</code></dt>
-	    <dd>A numeral indicating how well this candidate entity matches the query: a higher score indicates a better match;</dt>
+	    <dd>An optional numeral indicating how well this candidate entity matches the query: a higher score indicates a better match;</dt>
```

Comment:
Should the score be limited to a specific range in the spec ?
In the wikidata recon service scores are all bound to 0-100 range.
I don't have an opinion on this but it's the good place/time to discuss it.
The only input I can share is that having all the score on 0-100 range is clear and easy to interpret. But I don't know if some cases would need something different?

Reply:
Requiring normalization is tempting. The only other strong candidate for range would be 0-1.0.

Having said that, it opens a whole range of questions like:
- are scores comparable over time? for same search? for different searches? for different services?
- what does it mean to have a "perfect" score?

I think most search services punt on this and just say "bigger is better and you can't derive any additional information from scores," but, of course, users always try to infer some meaning anyway. Not sure what the best answer is...

Reply:
I'd leave the score range unspecified for now, to avoid getting roped up into an independent consideration. We can introduce a normalization in a separate PR if it is desired.